runway/cfngin/hooks/aws_lambda.py

```diff
@@ -401,7 +592,19 @@ def upload_lambda_functions(context, provider, **kwargs):
             build. Keys correspond to function names, used to derive key
             names for the payload. Each value should itself be a dictionary,
             with the following data:
-
+            **use_pipenv (Optional[bool])**:
+                Will determine if pipenv will be used to generate requirements.txt
+                from an existing Pipfile.
+            **pipenv_timeout (Optional[int])**
+                Time in milliseconds to wait while running pipenv.
```

Comment:
`Popen.communicate()` timeout is in seconds, not milliseconds.

> If the process does not terminate after timeout seconds, a TimeoutExpired exception will be raised. Catching this exception and retrying communication will not lose any output.

https://docs.python.org/3.7/library/subprocess.html#subprocess.Popen.communicate